src/test/java/com/microsoft/sqlserver/testframework/AbstractTest.java

```diff
@@ -145,8 +148,22 @@ public static void setup() throws Exception {
         }
 
         if (null == akvProvider) {
-            akvProvider = new SQLServerColumnEncryptionAzureKeyVaultProvider(applicationClientID, applicationKey);
-            map.put(Constants.AZURE_KEY_VAULT_NAME, akvProvider);
+            File file = null;
+            try {
+                file = new File("mssql-jdbc.properties");
```

Comment:
Why did we define MSSQL_JDBC_PROPERTIES and AKV_TRUSTED_ENDPOINTS_KEYWORD as constants and not use them?

Reply:
weird.